source-kafka@0.25.rb

```diff
@@ -0,0 +1,34 @@
+require 'fileutils'
+
+class SourceKafkaAT025 < Formula
+  v = "v0.25.0"
```

Comment:
Should this be v0.26.0?

Reply:
No, because that's the formula for version 0.25. 0.26 is in `source-kafka.rb`